React/Modules/RCTEventEmitter.m

```diff
@@ -78,21 +78,22 @@ - (void)dealloc
     RCTLogError(@"`%@` is not a supported event type for %@. Supported events are: `%@`",
                 eventName, [self class], [[self supportedEvents] componentsJoinedByString:@"`, `"]);
   }
-  if (_listenerCount == 0) {
+  _listenerCount++;
+  if (_listenerCount == 1) {
```

Comment:
that would trigger this every time you add a listener, while you want it to be triggered only the first time as far as I understood